data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones 1"
+	name "Neutralize broken Drone"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drone."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 20
+		"combat rating" > 200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking past a group of Navy soldiers, you hear one shout out: "Not again, another drone seems to have a malfunction and attacks ships in orbit. Who's turn is it to shoot it down?"`
+			`Nobody answers.`
+			`What do you want to do?`
+			choice
+				`	(Raise your hand and offer to shoot it down.)`
+					goto yes
+				`	(Ignore them and walk on.)`
+					decline
+			label yes
+			`You raise your hand and one of the soldiers notices you.`
+			`"You? Why not. Try not to hurt yourself."`
+			`You got the job.`
+				accept
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis marked
```

Comment:
About this `waiting`: It slightly confused me during the last days. I just sat down and figured out the problem. In "There Might Be Riots part 3B" they have this cool behavior where some of the drones take off with the player and since they are very close, they attack the player. This has two advantages: First it creates immediate action and second it generates two waves of attacks.
I wanted to duplicate this behavior, but apparently drones can't take of from planets with governments they don't like.
I'm not sure how to solve this, I'd love to have this behavior, but that's not possible if the drones dislike the republic. Though, I think it would make sense that the Republic ships help fighting the emerging AI drones since they could become a major threat.
Do you think, it would just be better to add another "support" government, that fights the drones while the drones are friendly against the republic (and everybody else)? That would also resolve the issue, that players could just wait until the other NPC's destroyed the drones.